conformance/lang/expressions/checking-expression/checkpanic_expr.balt

```diff
@@ -0,0 +1,1050 @@
+Test-Case: output
+Description: Test checkpanic-expr with basic types.
+Labels: int, float, decimal, string, nil-type, nil-literal, error-type, decimal, DecimalFloatingPointNumber, DecimalNumber,
+        FloatingPointTypeSuffix, HexFloatingPointLiteral, HexIntLiteral, checkpanic, function-defn, function-call-expr, return-stmt,
+        union-type, optional-type, int:MAX_VALUE, int:MIN_VALUE
+
+function init() {
+    io:println(fn1(0)); // @output 0
+    io:println(fn1(int:MAX_VALUE)); // @output 9223372036854775807
+    io:println(fn1(int:MIN_VALUE)); // @output -9223372036854775808
+    io:println(fn1(0x123Ffab1)); // @output 306182833
+    io:println(fn1(-0X123a)); // @output -4666
+
+    io:println(fn2(0.0001)); // @output 1.0E-4
+    io:println(fn2(-0.0001)); // @output -1.0E-4
+    io:println(fn2(0x1.21abc)); // @output 1.1315269470214844
+    io:println(fn2(121.121f)); // @output 121.121
+
+    io:println(fn3(1231d)); // @output 1231
+    io:println(fn3(12.12)); // @output 12.12
+    io:println(fn3(-123d)); // @output -123
+
+    io:println(fn4("0.000")); // @output 0.000
+    io:println(fn4("abc")); // @output abc
+
+    io:println(fn5(null)); // @output
+    io:println(fn5(())); // @output
+}
+
+function fn1(int x) returns error|int {
```

Comment:
Why do you need error in return type?

Reply:
Check other places also

Reply:
My bad. We don't need `error` type there. will update.

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-spec/pull/1069/commits/9bbe838a3d2b202e3159bf66cca24c931438633a and https://github.com/ballerina-platform/ballerina-spec/pull/1069/commits/13778f9e3e433f968993a7f7eacf616d851656f1